EU4toV2/Source/EU4World/World.cpp

```diff
@@ -212,6 +243,23 @@ void EU4::World::dropMinoritiesFromCountries()
 	}
 }
 
+void EU4::World::addTradeGoodsToCountries() const
```

Comment:
Shouldn't it rather be addTradeGoodsToProvinces()?

Reply:
thanx, fixed.